tests/desktop/test_details_page.py

```diff
@@ -125,7 +125,7 @@ def test_other_addons_label_when_there_are_multiple_authors(self, mozwebqa):
 
     @pytest.mark.nondestructive
     def test_other_addons_label_when_there_is_only_one_author(self, mozwebqa):
-        addon_with_one_authors = 'F1 by Mozilla Labs'
+        addon_with_one_authors = 'MemChaser'
```

Comment:
@stephendonner I am surprised you didn't also take the opportunity to change the variable name to `addon_with_one_author`. ;)